### DISCUSSION TITLE
Gemini api said that violate term

### DISCUSSION THREAD

**Discussion by phuclt9** [immersive-translate/immersive-translate#1728] [Help]
![Uploading Screenshot_20240520-123245.png…]()
### 概述 | Summary

I use immersive Translate with google gemini api. But after translate 1 - 2 page it say that gemini reach capacity limit i wait and try again but not thing change. In google AI studio they tell me that:" This project appears to be circumventing our quota restrictions via multiple projects acting as one."
Please help me!

**Comment by theowenyoung** [comment]
This is Gemini's response, unrelated to immersive translate, you can contact Gemini's staff to confirm the reason.

**Reply by theowenyoung** [reply]
To our knowledge, Gemini does not provide an API with the corresponding speed as it claims. You may purchase their service to use it.

**Reply by phuclt9** [reply]
Thank for your advice

**Comment by phuclt9** [comment]
Thank for your response

**Comment by phuclt9** [comment]
And one more question. If i subcrible pro member, does it include gemini
pro 1.5 (lasted) paid version?
